src/object_properties.cpp

```diff
@@ -128,7 +126,6 @@ void ObjectProperties::deSerialize(std::istream &is)
 
 	hp_max = readU16(is);
 	physical = readU8(is);
-	weight = readF32(is);
```

Comment:
Same here, you'll need to skip 32 bits. Not sure if there's a function for that

Reply:
`readF32(is);`